deps/db/src/logseq/db/frontend/property.cljs

```diff
@@ -277,7 +277,7 @@
                :hide? true}
       :queryable? false}
      ;; Task props
-     :logseq.task/status
+     :logseq.property/status
```

Comment:
I chose :logseq.property over :logseq.property.task because it's shorter and some of these properties may be useful for built-in classes e.g. deadline or scheduled